media-gfx/sane_backends/sane_backends-1.0.25.recipe

```diff
@@ -18,17 +18,16 @@ documentation.
 HOMEPAGE="http://www.sane-project.org"
 COPYRIGHT="David Mosberger-Tang, Andy Beck"
 LICENSE="GNU LGPL v2"
-REVISION="5"
-SOURCE_URI="https://alioth.debian.org/frs/download.php/file/4146/sane-backends-$portVersion.tar.gz"
+REVISION="6"
+SOURCE_URI="https://gitlab.com/sane-project/backends/uploads/68e5c0952bb81ecf0541c746fc1aa607/sane-backends-$portVersion.tar.gz"
 CHECKSUM_SHA256="a4d7ba8d62b2dea702ce76be85699940992daf3f44823ddc128812da33dc6e2c"
 SOURCE_DIR="sane-backends-$portVersion"
 PATCHES="
 	sane_backends-$portVersion.patchset
 	sane_disable_libusb_context.patchset
 	"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="x86_gcc2"
```

Comment:
Please restore ARCHITECTURES and SECONDARY_ARCHITECTURES.

Reply:
fixed